<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1240</width>
      <height>660</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>260</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>LogIn</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_1">
      <property name="geometry">
       <rect>
        <x>635</x>
        <y>226</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="pw_1">
      <property name="geometry">
       <rect>
        <x>635</x>
        <y>288</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID                  ||</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>283</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password      ||</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>487</x>
        <y>60</y>
        <width>270</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>25</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 25pt &quot;나눔고딕&quot;;</string>
      </property>
      <property name="text">
       <string>재무 관리 프로그램</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1240</width>
        <height>660</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 100, 255, 100), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_30</zorder>
     <zorder>pushButton</zorder>
     <zorder>id_1</zorder>
     <zorder>pw_1</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>460</width>
        <height>370</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>456</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>판매 총액 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="sale_sum">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>456</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="plus_sum">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>550</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="st_1">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>540</y>
        <width>60</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>590</y>
        <width>60</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="myselete">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>535</y>
        <width>61</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>485</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>조회 날짜 지정</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>재무 조회</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>456</y>
        <width>20</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>원</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>503</y>
        <width>20</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>원</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>590</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="st_2">
      <property name="geometry">
       <rect>
        <x>231</x>
        <y>540</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="ed_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>590</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="st_3">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>540</y>
        <width>40</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>545</y>
        <width>17</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>년</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>208</x>
        <y>595</y>
        <width>17</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>년</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>595</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>월</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>545</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>월</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>595</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>일</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>545</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>일</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>540</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>조회 시작일 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>조회 종료일 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>495</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 121, 0);
color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string>예) 1994.05.19</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>340</width>
        <height>370</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>550</y>
        <width>20</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>원</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>구입 총액 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="Purchase_sum">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>500</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>70</y>
        <width>401</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot_2" native="true">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>260</y>
        <width>401</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 100), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1240</width>
        <height>660</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 100, 255, 100), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>70</y>
        <width>463</width>
        <height>373</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>477</x>
        <y>70</y>
        <width>343</width>
        <height>373</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>827</x>
        <y>70</y>
        <width>403</width>
        <height>183</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>827</x>
        <y>260</y>
        <width>406</width>
        <height>183</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>483</y>
        <width>310</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>530</y>
        <width>310</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 233, 79);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>580</y>
        <width>310</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>550</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>이윤 총액 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>600</y>
        <width>20</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>원</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>600</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>조회 기간 내 이윤 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="plus_sum_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>600</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>630</y>
        <width>310</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(92, 53, 102);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>828</x>
        <y>450</y>
        <width>406</width>
        <height>183</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot_3" native="true">
      <property name="geometry">
       <rect>
        <x>831</x>
        <y>450</y>
        <width>401</width>
        <height>180</height>
       </rect>
      </property>
     </widget>
     <zorder>label_34</zorder>
     <zorder>label_33</zorder>
     <zorder>label_29</zorder>
     <zorder>label_28</zorder>
     <zorder>label_27</zorder>
     <zorder>label_23</zorder>
     <zorder>label_24</zorder>
     <zorder>label_26</zorder>
     <zorder>label_25</zorder>
     <zorder>label_22</zorder>
     <zorder>customPlot_2</zorder>
     <zorder>customPlot</zorder>
     <zorder>tableView_1</zorder>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
     <zorder>sale_sum</zorder>
     <zorder>plus_sum</zorder>
     <zorder>st_1</zorder>
     <zorder>ed_1</zorder>
     <zorder>myselete</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>ed_2</zorder>
     <zorder>st_2</zorder>
     <zorder>ed_3</zorder>
     <zorder>st_3</zorder>
     <zorder>label_6</zorder>
     <zorder>label_8</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_19</zorder>
     <zorder>tableView_2</zorder>
     <zorder>label_20</zorder>
     <zorder>label_21</zorder>
     <zorder>Purchase_sum</zorder>
     <zorder>label_7</zorder>
     <zorder>label_31</zorder>
     <zorder>label_32</zorder>
     <zorder>plus_sum_2</zorder>
     <zorder>customPlot_3</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>id_1</tabstop>
  <tabstop>pw_1</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>st_1</tabstop>
  <tabstop>st_2</tabstop>
  <tabstop>st_3</tabstop>
  <tabstop>ed_1</tabstop>
  <tabstop>ed_2</tabstop>
  <tabstop>ed_3</tabstop>
  <tabstop>myselete</tabstop>
  <tabstop>tableView_1</tabstop>
  <tabstop>tableView_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
